### ISSUE TITLE
Definition is not found when it's in the same file

### ISSUE THREAD

**Issue by eych** [henriiik/vscode-perl#8]
![example_pl_ _evo](https://user-images.githubusercontent.com/112840/34409696-529ae59e-ebc3-11e7-8128-365b361d2b44.png)

When the function is defined in the same file as the code that's using it, then Go to Definition always results in "No definition found" message.

**Comment by henriiik** [comment]
Hello!

When you save the file a .vstags file should be created in your project root. and the go to definition should work. Do you not get the .vstags file? and if you do is the function you are looking for not in it?

**Comment by eych** [comment]
I originally had this problem it on a really big Perl project where I created the file in my screenshot to show it. I've now tried to create the same example file on a new project and as you say `.vstags` file is created on saving and it jumps to the definition properly.

I've now deleted the `.vstags` file in the big project and tried to jump to definition again in the file where I had this problem and it now works fine. I think the issue can be closed, the problem is still there probably, but it's impossible for me to demonstrate it now.

Thanks!
